src/openzaak/components/catalogi/api/viewsets/relatieklassen.py

```diff
@@ -81,15 +82,15 @@ class ZaakTypeInformatieObjectTypeViewSet(
     }
 
     def get_concept(self, instance):
-        zaaktype = getattr(instance, "zaaktype", None) or self.get_object().zaaktype
+        ziot = self.get_object()
+        zaaktype = getattr(instance, "zaaktype", None) or ziot.zaaktype
         informatieobjecttype = (
-            getattr(instance, "informatieobjecttype", None)
-            or self.get_object().informatieobjecttype
+            getattr(instance, "informatieobjecttype", None) or ziot.informatieobjecttype
         )
-        return zaaktype.concept and informatieobjecttype.concept
+        return zaaktype.concept or informatieobjecttype.concept
```

Comment:
@sergei-maertens betekent dit dat je een `ZaakType-InformatieObjectType` moet kunnen verwijderen zelfs als of `zaaktype` of `informatieobjecttype` `concept=false` heeft?

Reply:
een ZIOT is concept (en dus verwijderbaar) zodra 1 van de relatie-elementen concept is